### ISSUE TITLE
Set Locale to UTF-8, But Warning Persists

### ISSUE THREAD

**Issue by nicolas-miari** [fastlane/fastlane#12501]
### New Issue Checklist

- [x] Updated fastlane to the latest version
- [x] I read the [Contribution Guidelines](https://github.com/fastlane/fastlane/blob/master/CONTRIBUTING.md)
- [x] I read [docs.fastlane.tools](https://docs.fastlane.tools)
- [x] I searched for [existing GitHub issues](https://github.com/fastlane/fastlane/issues)

### Issue Description

I keep getting the dreaded warning:

> WARNING: fastlane requires your locale to be set to UTF-8. To learn more go to https://docs.fastlane.tools/getting-started/ios/setup/#set-up-environment-variables

when I run `$ fastlane init` on my app's directory. 
As suggested here and there, I've already added the three lines:
 
    export LANG=en_US.UTF-8
    export LANGUAGE=en_US.UTF-8
    export LC_ALL=en_US.UTF-8

to my profile (`~/.bashrc`).

I'm positive this is the right profile, because `$ echo SHELL` gives:

> /bin/bash

**Perhaps I am missing something?**

---

##### Complete output when running fastlane, including the stack trace and command used

<details>
  <pre>[✔] 🚀 
[18:59:59]: WARNING: fastlane requires your locale to be set to UTF-8. To learn more go to https://docs.fastlane.tools/getting-started/ios/setup/#set-up-environment-variables
[19:00:00]: ------------------------------
[19:00:00]: --- Step: default_platform ---
[19:00:00]: ------------------------------

--------- ios---------
----- fastlane ios custom_lane
Description of what the lane does

Execute using `fastlane [lane_name]`
[19:00:00]: ------------------
[19:00:00]: fastlane is already set up at path `./fastlane/`, see the available lanes above
[19:00:00]: 
[19:00:00]: ------------------------------
[19:00:00]: --- Where to go from here? ---
[19:00:00]: ------------------------------
[19:00:00]: 📸  Learn more about how to automatically generate localized App Store screenshots:
[19:00:00]: 		https://docs.fastlane.tools/getting-started/ios/screenshots/
[19:00:00]: 👩‍✈️  Learn more about distribution to beta testing services:
[19:00:00]: 		https://docs.fastlane.tools/getting-started/ios/beta-deployment/
[19:00:00]: 🚀  Learn more about how to automate the App Store release process:
[19:00:00]: 		https://docs.fastlane.tools/getting-started/ios/appstore-deployment/
[19:00:00]: 👩‍⚕️  Learn more about how to setup code signing with fastlane
[19:00:00]: 		https://docs.fastlane.tools/codesigning/getting-started/
[19:00:00]: WARNING: fastlane requires your locale to be set to UTF-8. To learn more go to https://docs.fastlane.tools/getting-started/ios/setup/#set-up-environment-variables
</pre>
</details>

### Environment

<details>
  <pre><details><summary>🚫 fastlane environment 🚫</summary>

### Stack

| Key                         | Value                                                          |
| --------------------------- | -------------------------------------------------------------- |
| OS                          | 10.13.4                                                        |
| Ruby                        | 2.3.3                                                          |
| Bundler?                    | false                                                          |
| Git                         | git version 2.15.1 (Apple Git-101)                             |
| Installation Source         | /usr/local/bin/fastlane                                        |
| Host                        | Mac OS X 10.13.4 (17E202)                                      |
| Ruby Lib Dir                | /System/Library/Frameworks/Ruby.framework/Versions/2.3/usr/lib |
| OpenSSL Version             | LibreSSL 2.2.7                                                 |
| Is contained                | false                                                          |
| Is homebrew                 | false                                                          |
| Is installed via Fabric.app | false                                                          |
| Xcode Path                  | /Applications/Xcode.app/Contents/Developer/                    |
| Xcode Version               | 9.3.1                                                          |

### System Locale

| Error                       |
| --------------------------- |
| No Locale with UTF8 found 🚫 |

### fastlane files:

<details><summary>`./fastlane/Fastfile`</summary>

```ruby
# This file contains the fastlane.tools configuration
# You can find the documentation at https://docs.fastlane.tools
#
# For a list of all available actions, check out
#
#     https://docs.fastlane.tools/actions
#
# For a list of all available plugins, check out
#
#     https://docs.fastlane.tools/plugins/available-plugins
#

# Uncomment the line if you want fastlane to automatically update itself
# update_fastlane

default_platform(:ios)

platform :ios do
  desc "Description of what the lane does"
  lane :custom_lane do
    # add actions here: https://docs.fastlane.tools/actions
  end
end

```
</details>

<details><summary>`./fastlane/Appfile`</summary>

```ruby
# app_identifier("[[APP_IDENTIFIER]]") # The bundle identifier of your app
# apple_id("[[APPLE_ID]]") # Your Apple email address


# For more information about the Appfile, see:
#     https://docs.fastlane.tools/advanced/#appfile

```
</details>

### fastlane gems

| Gem      | Version | Update-Status |
| -------- | ------- | ------------- |
| fastlane | 2.95.0  | ✅ Up-To-Date  |

### Loaded fastlane plugins:

**No plugins Loaded**

<details><summary><b>Loaded gems</b></summary>

| Gem                       | Version      |
| ------------------------- | ------------ |
| did_you_mean              | 1.0.0        |
| slack-notifier            | 2.3.2        |
| atomos                    | 0.1.2        |
| claide                    | 1.0.2        |
| colored2                  | 3.1.2        |
| nanaimo                   | 0.2.5        |
| xcodeproj                 | 1.5.7        |
| rouge                     | 2.0.7        |
| xcpretty                  | 0.2.8        |
| terminal-notifier         | 1.8.0        |
| unicode-display_width     | 1.3.2        |
| terminal-table            | 1.8.0        |
| plist                     | 3.4.0        |
| public_suffix             | 2.0.5        |
| addressable               | 2.5.2        |
| multipart-post            | 2.0.0        |
| word_wrap                 | 1.0.0        |
| tty-screen                | 0.6.4        |
| tty-cursor                | 0.5.0        |
| tty-spinner               | 0.8.0        |
| babosa                    | 1.0.2        |
| colored                   | 1.2          |
| highline                  | 1.7.10       |
| commander-fastlane        | 4.4.6        |
| excon                     | 0.62.0       |
| faraday                   | 0.15.1       |
| unf_ext                   | 0.0.7.5      |
| unf                       | 0.1.4        |
| domain_name               | 0.5.20180417 |
| http-cookie               | 1.0.3        |
| faraday-cookie_jar        | 0.0.6        |
| fastimage                 | 2.1.3        |
| gh_inspector              | 1.1.3        |
| mini_magick               | 4.5.1        |
| multi_json                | 1.13.1       |
| multi_xml                 | 0.6.0        |
| rubyzip                   | 1.2.1        |
| security                  | 0.1.3        |
| xcpretty-travis-formatter | 1.0.0        |
| dotenv                    | 2.4.0        |
| bundler                   | 1.16.1       |
| faraday_middleware        | 0.12.2       |
| naturally                 | 2.1.0        |
| simctl                    | 1.6.4        |
| uber                      | 0.1.0        |
| declarative               | 0.0.10       |
| declarative-option        | 0.1.0        |
| representable             | 3.0.4        |
| retriable                 | 3.1.1        |
| mime-types-data           | 3.2016.0521  |
| mime-types                | 3.1          |
| little-plugger            | 1.1.4        |
| logging                   | 2.2.2        |
| jwt                       | 2.1.0        |
| memoist                   | 0.16.0       |
| os                        | 0.9.6        |
| signet                    | 0.8.1        |
| googleauth                | 0.6.2        |
| httpclient                | 2.8.3        |
| google-api-client         | 0.13.6       |
| emoji_regex               | 0.1.1        |
| io-console                | 0.4.6        |
| CFPropertyList            | 3.0.0        |
| libxml-ruby               | 2.9.0        |
| json                      | 2.1.0        |
</details>

*generated on:* **2018-05-11**
</details></pre>
</details>

**Comment by nicolas-miari** [comment]
I fail to see how it can be related to code signing. I was just setting up Fastlane, there is no existing lane that requires code signing.
I'm setting up CircleCI, and since it uses Fastlane scan for build/testing, I figured I'd go the extra mile and add support for it directly in my project's repository...

**Comment by Aquima** [comment]
@nicolas-miari 
you set a Environment variable  parameter and set a like a jenkins i'dont use CircleCi but i believe is the same problem 
https://1drv.ms/u/s!AkJPOBbhyZlwgopbKen4KzogsR11Rg

**Comment by nicolas-miari** [comment]
@Aquima  
Yes, I have successfully silenced the warning on CircleCI using configuration like this:

    version: 2
    jobs:
      build_and_test:
        macos:
          xcode: "9.3.0"
        environment:
          LC_ALL: en_US.UTF-8
          LANG: en_US.UTF-8

However, I can not achieve the same when running fastlane on my local machine, (e.g. `$ fastlane init`), even though I have exported the environment variables in my bash profile.

**Comment by Aquima** [comment]
edit and use the next command
# nano .bashrc 
or 
# nano .profile 
you find this files using 
# ls  ~/.bashrc ~/.profile
and add  
export LANG=en_US.UTF-8
export LANGUAGE=en_US.UTF-8
export LC_ALL=en_US.UTF-8
and later press ctrl+x and late save changes

**Comment by nicolas-miari** [comment]
@Aquima 
Yes, that is **exactly** what I did in the first place (see my original post opening the issue).
However, the changes didn't seem to take effect until I ran this command:

    $ source ~/.bashrc

**Comment by Aquima** [comment]
@nicolas-miari  check this video 
https://1drv.ms/v/s!AkJPOBbhyZlwgopccv66ZAGbCCVuiQ
